src/shared/components/studyTagsTooltip/StudyTagsTooltip.tsx

```diff
@@ -38,12 +41,14 @@ export type StudyTagsTooltipProps = {
     mouseEnterDelay: number;
     placement: string;
     children: any;
+    infoIcon: boolean;
```

Comment:
Better to replace this by type references e.g. INFO_ICON, LOCK_ICON.

Reply:
Fixed